static/simple-portfolio-template/index.html

```diff
@@ -0,0 +1,344 @@
+          <p class="circle-text">
+            Interface <br />
+            Design
+          </p>
+        </div>
+
+        <div
+          class="about-me-circle absolute bottom-5 md:bottom-auto md:right-5"
+        >
+          <p class="circle-text">
+            Mobile <br />
+            Apps
+          </p>
+        </div>
+      </article>
+
+      <span class="h-40 w-px bg-slate-800"></span>
+
+      <article class="my-5 flex w-2/3 flex-col items-center">
+        <h3
+          class="relative text-3xl font-semibold after:absolute after:-bottom-0.5 after:left-1/4 after:right-1/4 after:h-0.5 after:bg-yellow-400"
+        >
+          About
+        </h3>
+
+        <p class="my-5 mt-14 text-center text-base">
+          It is a long established fact that a reader will be distracted by the
+          readable content of a page when looking at its layout. The point of
+          using LoremIt is a long established fact that a reader will be
+          distracted by the readable content of a page when looking at its
+          layout. The point of using Lorem
+        </p>
+      </article>
+
+      <article
+        class="flex w-2/3 flex-col items-center justify-center border border-dashed border-slate-800"
+      >
+        <img src="./assets/images/about-img.jpg" alt="About Me" class="w-1/2" />
+      </article>
+
+      <article class="relative mt-10">
+        <a href="" class="read-more-text text-black after:bg-black">
+          Read More
+        </a>
+      </article>
+    </section>
+
+    <section class="flex flex-col items-center py-14" id="projects-section">
+      <h3 class="section-title">My Projects</h3>
+
+      <article
+        class="grid-cols-fluid my-20 grid w-5/6 place-items-center gap-8 lg:w-3/4"
+      >
+        <div>
+          <img src="./assets/images/p-1.jpg" alt="Project 1" />
+        </div>
+        <div>
+          <img src="./assets/images/p-2.jpg" alt="Project 2" />
+        </div>
+        <div>
+          <img src="./assets/images/p-3.jpg" alt="Project 3" />
+        </div>
+
+        <div>
+          <img src="./assets/images/p-4.jpg" alt="Project 4" />
+        </div>
+        <div>
+          <img src="./assets/images/p-5.jpg" alt="Project 5" />
+        </div>
+        <div>
+          <img src="./assets/images/p-6.jpg" alt="Project 6" />
+        </div>
+
+        <div>
+          <img src="./assets/images/p-7.jpg" alt="Project 7" />
+        </div>
+        <div>
+          <img src="./assets/images/p-8.jpg" alt="Project 8" />
+        </div>
+        <div>
+          <img src="./assets/images/p-9.jpg" alt="Project 9" />
+        </div>
+      </article>
+    </section>
+
+    <section
+      class="mt-10 flex flex-col items-center py-0 md:flex-row"
+      id="expands-section"
+    >
+      <span class="my-10 h-px w-full bg-slate-800/30 md:hidden"></span>
+
+      <article class="w-full md:w-1/2">
+        <img
+          src="./assets/images/expand-img.jpg"
```

Comment:
You can export vector images from Figma in SVG format, they don't lose quality when scaled and also have smaller file sizes.  I can show you how to do that if you need help.

![image](https://user-images.githubusercontent.com/6149523/177935706-408c0b5e-d4ad-4f4f-a4b5-3a2e28405ac7.png)

Reply:
Excellent point, but the template I downloaded came with the images as such, so in this case, I have to stick with this.